tests/test_wave_05.py

```diff
@@ -71,29 +70,37 @@ def test_create_goal(client):
         }
     }
 
-@pytest.mark.skip(reason="test to be completed by student")
 def test_update_goal(client, one_goal):
-    pass
     # Act
+    response = client.put("/goals/1", json={
+        "title":"Updated goal title",
+    })
+    response_body = response.get_json()
     # ---- Complete Act Here ----
 
     # Assert
     # ---- Complete Assertions Here ----
-    # assertion 1 goes here
-    # assertion 2 goes here
-    # assertion 3 goes here
+    assert response.status_code ==200
+    assert "goal" in response_body
+    assert response_body == {
+        "goal": {
+        "id":1,
+        "title":"Updated goal title",
+    }}
     # ---- Complete Assertions Here ----
```

Comment:
For completeness, we should query the database again and make sure that the goal with this id has been updated in the database.